package.json

```diff
@@ -10,12 +10,11 @@
     "build": "src/*.js"
   },
   "scripts": {
-    "build": "fedx-scripts webpack",
+    "build": "make build",
```

Comment:
Is there not a fedex-script that does what we want? Is this kind of build something we would want to move into fedx-scripts?

Reply:
good point yeah. I think we would have to put some thought there indeed. I find that the published recipe there is only designed for apps, hence I had to discover stuff like this by looking at frontend-platform etc. I think there is room for us to publish library friendly recipe in frontend-build... or at least a doc recipe if not code recipe

This model was from frontend-platform but I agree a fedx-scripts library-build or something would be better